src/components/label/index.tsx

```diff
@@ -0,0 +1,25 @@
+import { Component, h, Prop } from "@stencil/core"
+
+@Component({
+	tag: "smoothly-label",
+	styleUrl: "style.css",
+	scoped: true,
+})
+export class SmoothlyLabel {
+	@Prop() color: string
+	@Prop() name: string
+	@Prop() description: string
+	@Prop({ reflect: true }) shape: "rectangle" | "rounded"
+
+	render() {
+		return (
+			<div
+				title={this.description}
+				style={{
```

Comment:
i would personally say its preferable to define a CSS variable rather than applying a rule directly. applying a style like this makes it very difficult to override with other CSS rules

Reply:
idk how to do that

Reply:
index.tsx
```tsx
<Host style={{"--hue": this.hue }}>
  ...
</Host>
```
style.css
```css
:host {
  background-color: hue(var(--hue), 85% 70%);
}
```

Reply:
But I personally think the whole component should just use the `Color` type in smoothly.
The `common.css` will set the color accodingly: https://github.com/utily/smoothly/blob/master-1.0/src/assets/style/common.css#L21

So set it like this:
index.tsx
```tsx
@Prop({reflect: true}) color: Color = "default"
```
style.css
``` css
:host {
  background-color: rgb(var(--smoothly-color));
  color: rgb(var(--smoothly-color-contrast));
}
```

Reply:
\+

Reply:
what Magnus said. he was faster :upside_down_face: But using smoothly colours will not be sufficient probably. Needs more colours and its probably not in interest to style colours like these with keywords like "danger" and "success". maybe change name to hue so no one mixes it with the smoothly color property.